recipe/meta.yaml

```diff
@@ -11,8 +11,10 @@ source:
     - patches/Makefile.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage("lmdb", max_pin="x.x") }}
```

Comment:
I'm not even sure it's that. I'd go with patch-level (or exact pins) unless we have good evidence to the contrary.

Reply:
ABI laboratory is normally enough evidence. lmdb doesn't look like it will be getting more releases soon.

Reply:
It's missing the last 5 versions, but if you feel confident that lmdb didn't break ABI in the meantime - fine by me.